.github/workflows/test.yaml

```diff
@@ -13,15 +13,15 @@ jobs:
       - name: Setup buf
         uses: wizhi/setup-buf@v1
         with:
-          version: 1.1.0
+          version: 1.8.0
       - name: Checkout repository
         uses: actions/checkout@v2.3.4
       - name: install tools
         run: make install-tools
       - name: Setup golangci-lint
         uses: golangci/golangci-lint-action@v3.1.0
         with:
-          version: v1.45.0
+          version: v1.49.0
```

Comment:
@GGabriele is this required? Won't make install-tools take care of setting up golangci-lint?